integration_tests/lpdraw/draw.py

```diff
@@ -0,0 +1,100 @@
+from lpython import i32, f64, TypeVar
+import numpy
+
+H = TypeVar("H")
+W = TypeVar("W")
+
+def Pixel(H: i32, W: i32, Screen: i32[H, W], x: i32, y: i32) -> None:
+    Screen[y, x] = 255
+
+def Clear(H: i32, W: i32, Screen: i32[H, W]):
+    i: i32
+    j: i32
+    for i in range(H):
+        for j in range(W):
+            Screen[i, j] = 0
+
+def DisplayTerminal(H: i32, W: i32, Screen: i32[H, W]):
```

Comment:
Add a function to check if the pixels in `Screen` array as expected (using asserts). Like you can add a moderately strong test by summing up all the pixels in `Screen` array and then checking that the sum value is as expected.

Reply:
Done. Thank you so much for the guidance!

Reply:
Sorry, not yet done. Facing some issue with pushing the changes.